config/src/config.rs

```diff
@@ -824,6 +825,9 @@ pub struct Config {
     #[dynamic(default)]
     pub treat_east_asian_ambiguous_width_as_wide: bool,
 
+    #[dynamic(default)]
+    pub cellwidths: Option<Vec<CellWidth>>,
```

Comment:
Let's use an underscore here to separate the two words

```suggestion
    pub cell_widths: Option<Vec<CellWidth>>,
```